packages/wdio-junit-reporter/tests/reporter.test.js

```diff
@@ -72,21 +82,49 @@ describe('wdio-junit-reporter', () => {
         reporter.suites = suitesLog
 
         // verifies the content of the report but omits format by stripping all whitespace and new lines
-        expect(reporter.prepareXml(runnerLog).replace(/\s/g, '')).toBe(testLog.toString().replace(/\s/g, ''))
+        expect(reporter.buildJunitXml(mochaRunnerLog).replace(/\s/g, '')).toBe(testLog.toString().replace(/\s/g, ''))
+    })
+
+    it('generates xml output (Cucumber-style)', () => {
+        reporter.suites = featuresLog
+
+        // verifies the content of the report but omits format by stripping all whitespace and new lines
+        expect(reporter.buildJunitXml(cucumberRunnerLog).replace(/\s/g, '')).toBe(testCucumberLog.toString().replace(/\s/g, ''))
```

Comment:
Can we use `toMatchSnapshot()` for all the kind of assertion as it allows better insights for when it fails.

Reply:
Done :+1: